src/Php70/bootstrap.php

```diff
@@ -20,10 +20,13 @@
 }
 
 if (!function_exists('intdiv')) {
-    function intdiv($dividend, $divisor) { return p\Php70::intdiv($dividend, $divisor); }
+    function intdiv($num1, $num2) { return p\Php70::intdiv($num1, $num2); }
 }
 if (!function_exists('preg_replace_callback_array')) {
-    function preg_replace_callback_array(array $patterns, $subject, $limit = -1, &$count = 0) { return p\Php70::preg_replace_callback_array($patterns, $subject, $limit, $count); }
+    /**
+     * $flags is not supported.
+     */
+    function preg_replace_callback_array(array $pattern, $subject, $limit = -1, &$count = 0, $flags = null) { return p\Php70::preg_replace_callback_array($pattern, $subject, $limit, $count); }
```

Comment:
PHPs `preg_replace_callback_array` has a $flags argument, but the polyfill doesn't. so I added it. this would work the same.

Adding support for $flags is unrelated to this PR.